2023-07-monthly-report.org

```diff
@@ -0,0 +1,379 @@
+                        visibility: pub                                                                                                                                      
+                        unsafety: 0                                                   
+                        name: Sized                                                                                                                                          
+                        generic_params {                                                                                                                                     
+                                TypeParam [                                                                                                                                  
+                                        outer_attr:                                                                                                                          
+                                        type_representation: Self                                                                                                            
+                                        type_param_bounds: empty                                                                                                             
+                                        type: none                                                                                                                           
+                                ] // TypeParam                                                                                                                               
+                        } // generic_params 
+                        type_param_bounds: empty
+                        where clause: none
+                        trait_items: empty
+                ] // Trait
+                TupleStruct [
+                        mapping: [C: 0 Nid: 20 Hid: 29 Lid: 8]
+                        outer_attributes: empty
+                        visibility: private 
+                        struct_name: GenericStruct
+                        generic_params {
+                                TypeParam [ 
+                                        outer_attr: 
+                                        type_representation: T
+                                        type_param_bounds: empty
+                                        type: none
+                                ] // TypeParam
+                        } // generic_params 
+                        where_clause: none
+                        fields {
+#+END_SRC
+
+We will be making changes to this going forward but having a starting point gives us the chance to move forward.
+
+*** More error codes
+
+This month saw us adding these error codes to our diagnostics:
+
+1. E0572
+2. E0571
+3. E0268
+4. E0045
+5. E0323
+6. E0423
+7. E0133
+8. E0658
+9. E0093
+10. E0124
+11. E0070
```

Comment:
There are some error codes, which are being mentioned in above PR's but not here:

```rust
12. E0063
13. E0045
14. E0053
15. E0046
16. E0061
17. E0229
18. E0034

Reply:
Feel free to add them @MahadMuhammad

Reply:
I'll raise the PR when this PR(https://github.com/Rust-GCC/Reporting/pull/102) gets merged or should I raise the PR with all the changes (including new changes) in this file?

Reply:
I added them @MahadMuhammad